src/services/shuffel.js

```diff
@@ -0,0 +1,7 @@
+export default function shuffle(list, randomizer, seed) {
+  for (let i = list.length - 1; i > 0; i--) {
+    randomizer.seed(seed + i)
```

Comment:
Why do you reseed on every step? Why not pass a correctly seed RNG to the function and get rid of the seed argument?

Reply:
I'll talk to Jerry about this, thank you!